### DISCUSSION TITLE
Adjustable Countdown per Host

### DISCUSSION THREAD

**Discussion by steilerDev** [seriousm4x/UpSnap#892] [Feature requests]
The tool is great and easy to use - would it be possible to adjust the time of the "Wake" and "Shutdown" timers in the UI? 
E.g. if I know that my PC will boot in under  30 seconds, no need to show a 2 minute timer.

**Comment by tecnoprogram-thomas** [comment]
If I may add, I have some PC's that for some reasons only wake after the second request ... having to wait 2 minutes is really annoying, I would like to setup a lower timeout , or to force a new WOL request even if the countdown is not ended!
Great tool!

**Comment by OliverRhyme** [comment]
how can i disable the countdown?

**Comment by nathandaven** [comment]
+1, would love to see this! I know how long the PC takes for each action, so would like to manually set.
